app/controllers/internal/configs_controller.rb

```diff
@@ -139,4 +140,11 @@ def onboarding_params
       suggested_users
     ]
   end
+
+  def job_params
+    %i[
+      jobs_url
+      display_jobs_before_search
```

Comment:
This variable name with the `before_search` in it confused me a little and made me overthink what this variable is doing.  Maybe we can just rename it to display_jobs_banner or display_jobs_notice, if the before_search isn't too important? We describe where the job notice sits on the page in the site config variable description. You can ignore this if you don't feel the same though.

Reply:
100% I like that better, I was coming up so blank for that variable lol